MerlinAU.sh

```diff
@@ -1731,8 +1757,11 @@ _CreateDirectory_()
         "$inMenuMode" && _WaitForEnterKey_ "$mainMenuReturnPromptStr"
         return 1
     fi
-    # Clear directory in case any previous files still exist #
-    rm -f "${1}"/*
+    if [ ManualUpdateTrigger = "1" ]
```

Comment:
**Line** **1760**:
The Linter tool flagged this conditional statement as a potential syntactical error because it's comparing 2 different static strings which makes the comparison always "**FALSE**" while it was likely meant to have a variable reference; then there was another flag next to it saying it could be a potential source of "parameter not found."

Reply:
Great catch! I totally missed the $ symbol!